### PR TITLE
Belt resprite with soul? Blasphemy!

### PR DESCRIPTION
**Pull request by Youtubeboy139**
## About The Pull Request

The classic belts have been a staple of SS13 as a whole for years, all the way back since /TG was first created. They have pooled downstream to many other smaller servers with little to no change (except for the rare few) to their overall design. This PR aims to give the old belts some fresh life while retaining the soul that makes them that staple. Better shading, more defined shape/color, just an overall facelift.

Screenshot 2023-06-06 231153

Screenshot 2023-06-06 224628

Link to OLD belts for comparison 
https://cdn.discordapp.com/attachments/326849939240124419/1116113448259031040/image.png

## Why It's Good For The Game

Replaces a ~~depreciated~~ classic sprite with a better, more modern sprite.

## Changelog

:cl:
add: Added new EMT belt (really just renamed the Paramedic belt path)
image: Added new EMT belt sprite
image: Added shiny new belts and detective/syndicate holsters (`onmob` sprites included)
image: Removed old, unused red military belt sprite
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -90,7 +90,7 @@
 /obj/item/storage/belt/utility/chief/full/PopulateContents()
 	SSwardrobe.provide_type(/obj/item/screwdriver/power, src)
 	SSwardrobe.provide_type(/obj/item/crowbar/power, src)
-	SSwardrobe.provide_type(/obj/item/weldingtool/experimental, src)//This can be changed if this is too much //It's been 5 years
+	SSwardrobe.provide_type(/obj/item/weldingtool/experimental, src)//This can be changed if this is too much //It's been 5 years //Possibly longer
```
Comment (san7890) [inline code/game/objects/items/storage/belt.dm:93]:
```suggestion
	SSwardrobe.provide_type(/obj/item/weldingtool/experimental, src)
```

it's time

Comment (san7890) [reply code/game/objects/items/storage/belt.dm:93]: i know you wanted to retain sovl in this PR but some things are no longer meant to last. if you want to leave it as is just remove your `//Possibly longer` append and we'll just move on

**Review by Wallemations — APPROVED** [review]

File: code/game/objects/items/storage/belt.dm (hunk 1 above)
Comment (Youtubeboy139) [reply code/game/objects/items/storage/belt.dm]: I understand.

Rest in peace.

**Comment by Loafin34** [comment]
not a fan of botany belt, should be yellow. otherwise, looks like some good soul.

**Comment by Youtubeboy139** [comment]
> not a fan of botany belt, should be yellow. otherwise, looks like some good soul.

Screenshot 2023-06-07 013251

**Comment by Rex9001** [comment]
Great stuff, love the new sprites Keeping the old spirit but updating them

**Comment by MTandi** [comment]
> > not a fan of botany belt, should be yellow. otherwise, looks like some good soul.
> 
> Screenshot 2023-06-07 013251

Why yellow? Yellow is an engineering department color. I liked it more before because it looked like botany gloves.

**Comment by Singul0** [comment]
a positively received sprite PR? guh?!?

**Comment by Youtubeboy139** [comment]
> While you're here....... I've always wished the security belt was the same shape as the other belts.........
> 
> But really these look beautiful. The holsters especially are holy-moly!!!
> 
> One smallllll inconsistency to note though: all the belts have those pin-things the same height except the medical one, which is 1px shorter image

This is on purpose to better separate the Utility belts from the Specialty belts. If you notice the pack on them is smaller too

**Comment by Youtubeboy139** [comment]
> > > not a fan of botany belt, should be yellow. otherwise, looks like some good soul.
> > 
> > 
> > Screenshot 2023-06-07 013251
> 
> Why yellow? Yellow is an engineering department color. I liked it more before because it looked like botany gloves.

More people want the brown over the yellow. Brown wins, yellow is reverted.

**Comment by CRITAWAKETS** [comment]
Y'know I didn't expect much but this blew away those expectations, that looks like proper leather!

The colors on the CE's belt are kinda bad but that's more from the fact that the CE just has terrible colors to begin with.

**Comment by Youtubeboy139** [comment]
> Y'know I didn't expect much but this blew away those expectations, that looks like proper leather!
> 
> The colors on the CE's belt are kinda bad but that's more from the fact that the CE just has terrible colors to begin with.

I did my best with what I had available. That's all I can say.

**Comment by TheVekter** [comment]
What is this bullshit? A sprite PR people actually like?

Actually, this gives me a chance to request a change that's bugged the shit out of me for years - the champion belt really should have a brown strap and gold buckle, not be full gold. What title belt do you know that is completely gold?

image

I'm not sure how you'd do it, but you could even possibly break it up a little to show the metal segments like on real belts. Hell, maybe make the center part bigger and put a tiny "N" on it for Nanotrasen?

I dunno, I've always wanted it to look like an actual title belt, not just a fully gold one. Soul be damned.

**Comment by Youtubeboy139** [comment]
> What is this bullshit? A sprite PR people actually like?
> 
> Actually, this gives me a chance to request a change that's bugged the shit out of me for years - the champion belt really should have a brown/black strap and gold buckle, not be full gold. What title belt do you know that is completely gold?
> 
> image
> 
> I'm not sure how you'd do it, but you could even possibly break it up a little to show the metal segments like on real belts. Hell, maybe make the center part bigger and put a tiny "N" on it for Nanotrasen?
> 
> I dunno, I've always wanted it to look like an actual title belt, not just a fully gold one. Soul be damned.

I'mma be real with you chief, out of all these, I kinda phoned in the champion belt cause I was starting to lose my mind a bit. I might give it a pass later down the line when the PR is merged, but for now I just want to get this PR merged

**Comment by ExcessiveUseOfCobblestone** [comment]
medical is wearing white already, is it discernable with their normal outfit on?

**Comment by Youtubeboy139** [comment]
> medical is wearing white already, is it discernable with their normal outfit on?
Without Belt
Screenshot 2023-06-07 154758
With Belt
Screenshot 2023-06-07 154811

**Comment by TheVekter** [comment]
> > medical is wearing white already, is it discernable with their normal outfit on?
> 
> Without Belt Screenshot 2023-06-07 154758 With Belt Screenshot 2023-06-07 154811
> 
> If this is truly an issue I can make a revision, I still have the sprite bases all neatly saved, it'd be as simple as creating a new palette and recoloring it
> 
> EDIT: Like so Screenshot 2023-06-07 160307

I like that a lot better actually.

**Comment by Wallemations** [comment]
did you make all the changes you want or are you still working on this?
If you're still working on it, could you draft this? I don't wanna merge it if you're not done with it.

**Comment by Youtubeboy139** [comment]
> did you make all the changes you want or are you still working on this? If you're still working on it, could you draft this? I don't wanna merge it if you're not done with it.

No, I'm done with this one. Any future changes will be done in a separate PR.

**Review by Wallemations — APPROVED** [review]

**Pull request merged**